examples/common.cpp

```diff
@@ -271,6 +271,12 @@ bool gpt_params_parse(int argc, char ** argv, gpt_params & params) {
             params.use_color = true;
         } else if (arg == "--mlock") {
             params.use_mlock = true;
+        } else if (arg == "--gpu_layers") {
```

Comment:
suggest gpu-layers to align with other options.

Reply:
I don't understand what you mean.

Reply:
He's suggesting you replace the underscore with a hyphen, I think.

Reply:
Horenbergerb is right. sorry didn't make it explicit. as other parameters are all using hyphen.

Reply:
They are not all using it consistently, but it is a different issue.